helm/ida-auth/values.yaml

```diff
@@ -113,15 +113,16 @@ resources:
   # resources, such as Minikube. If you do want to specify resources, uncomment the following
   # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
   limits:
-    cpu: 4500m
-    memory: 4000Mi
+    cpu: '2'
+    memory: 5000Mi
   requests:
     cpu: 100m
     memory: 1000Mi
 additionalResources:
   ## Specify any JAVA_OPTS string here. These typically will be specified in conjunction with above resources
   ## Example: java_opts: "-Xms500M -Xmx500M"
-  javaOpts: "-Xms2000M -Xmx2000M"
+  javaOpts: "-Xms3000M -Xmx3000M"
```

Comment:
javaOpts: "-Xms3200M -Xmx3200M" and
  repository: mosipid/authentication-service
  tag: 1.3.0-beta.2 change to   repository: mosipqa/authentication-service
  tag: 1.3.x